kickstarts/centos8_build_machine.ks

```diff
@@ -7,10 +7,10 @@ zerombr
 clearpart --all --initlabel --drives=sda
 partition /boot --ondisk=sda --asprimary --size=1024 --fstype=xfs
 partition pv.1 --ondisk=sda --asprimary --size=10240 --grow
-volgroup kegerator8 pv.1
-logvol swap --vgname=kegerator8 --name=swap --size=8192
-logvol / --vgname=kegerator8 --name=root --size=20480 --fstype=xfs
-logvol /build --vgname=kegerator8 --name=build --size=20480 --fstype=xfs --grow
+volgroup kegerator pv.1
```

Comment:
This name is so silly haha - can we pick any volgroup name or does it have to be this?

Reply:
I decided to remove changes to this file from this PR